deb-arm/Dockerfile

```diff
@@ -112,6 +123,23 @@ RUN if [ "$DD_TARGET_ARCH" = "armhf" ] ; then \
 ENV PATH="/usr/local/go/bin:${PATH}"
 ENV PATH="${GOPATH}/bin:${PATH}"
 
+# Rust is needed to compile the SDS library
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
What is SDS?

Reply:
Here's a link to the [Sensitive Data Scanner](https://docs.datadoghq.com/sensitive_data_scanner/?tab=addscanningrulefromthelibraryrules) doc.